.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Test OSX and Linux
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: install
+        run: |
+          npm i
+      - publish:
+        run: |
+          npm run prebuild && npm run prebuild:upload -u ${PREBUILD_UPLOAD}
+        env:
+          CRATES_IO_TOKEN: ${{ secrets.PREBUILD_UPLOAD }}
```

Comment:
This can probably be removed

Reply:
How do you publish to npm? Manually?

Reply:
Yes, https://github.com/Razzeee/tree-sitter-elm/blob/master/HOW_TO_RELEASE.MD

I recently changed that for the language server, haven't really took the time to do something similar here.
https://github.com/elm-tooling/elm-language-server/blob/master/.github/workflows/npm-publish.yml